src/components/Badge/Badge.tsx

```diff
@@ -23,15 +23,14 @@ class Badge extends React.Component<BadgeProps, BadgeState> {
       leftWidth: 0,
       rightWidth: 0
     };
-    this.updateTextLength = this.updateTextLength.bind(this);
   }
   componentDidMount() {
     this.updateTextLength();
   }
   componentDidUpdate() {
     this.updateTextLength();
   }
-  updateTextLength() {
+  updateTextLength = () => {
```

Comment:
This is interesting, so, declaring the function object then simplify the binding declaration ?
I'm just learning, the preliminar examples I saw in the react site used all the bind(this) syntax.

Reply:
Yes it's a subtlety in typescript, this was discussed here https://github.com/kiali/kiali-ui/pull/179 and I think there was also a topic on kiali-dev.

Reply:
Nice. +1